src/index.js

```diff
@@ -23,8 +24,10 @@ class _App extends Component{
     return (
       <HashRouter>
         <Route component={Nav} />
+        
         <Route path='/' component={Home} exact />
-        <Route path='/users' component={Users} />
+        <Route exact path='/users' component={Users} />
+        <Route path='/users/:id' component={User} />
```

Comment:
looks great! we just might get an error 'cannot find module User' until we build the User.js component

Reply:
Yes, I have many ideas in my head one of it how to add items to the user, I thought I can finish it before 3 am but I facing some issues during work and take time to fix it, and leave some behind and you catch it :) Thank you Nikki